docs/source/layouts.rst

```diff
@@ -3,4 +3,139 @@ Layouts
 =======
 
 
+Glitter templates render a list of columns in a Django template, with each Glitter model having a
+defined number of layouts.
 
+
+Quick example
+=============
+
+This example layout defines a ``Default`` layout with 3 columns, ``content``, ``sidebar`` and
+``footer``, registered to the Glitter Pages model.
+
+.. code:: python
+
+    from glitter import columns, templates
+    from glitter.layouts import PageLayout
+
+
+    @templates.attach('glitter_pages.Page')
+    class Default(PageLayout):
+        content = columns.Column(width=640)
+        sidebar = columns.Column(width=320)
+        footer = columns.Column(width=960)
+
+Which works with the following template ``glitter/default.html``.
+
+.. code:: html
+
+    <main>
+        <aside>
+            {{ columns.sidebar }}
+        </aside>
+        <article>
+            {{ columns.content }}
+        </article>
+    </main>
+    <footer>
+        {{ columns.footer }}
+    </footer>
+
+
+Discovery of layouts
+====================
+
+Glitter will automatically look for a ``layouts`` module in each application and imports it. This
+allows you to keep your layouts in your project in a well known and consistent location -
+``layouts.py``.
+
+
+Columns
+=======
+
+The most important part of a layout - the columns of content which will be rendered into the
+template. Columns are specified by class attributes.
+
+Example:
+
+.. code:: python
+
+    from glitter import columns
+    from glitter.layouts import PageLayout
+
+
+    class Default(PageLayout):
+        content = columns.Column(width=640)
+
+
+Attributes
+----------
+
+:attr:`width`
```

Comment:
This rendering weird can we just have


**_width_**
         
     The width of the column in pixels. lore ipsum